task-1/question-1.js

```diff
@@ -0,0 +1,17 @@
+//question: 
+// 1)Write a   program which reads a string from standard input reverses the string and writes it to the standard output.
+// a)The program should run in a   
+// stand-by mode and should not be terminated after reversing the first string.
+
+
+// solution:
+var stdin = process.stdin;
+stdin.on('data',(data)=>{
+    if(data.toString() === 'exit\r\n') process.exit();
+    else
+    {
+    const output = data.toString().split('').reverse().join('');
+    console.log(output);
+    }
+})
```

Comment:
May be you can consider formatting this file, so that the `if else` blocks will have proper indentation

Reply:
done